includes/template-library/sources/remote.php

```diff
@@ -65,19 +71,19 @@ public function register_data() {}
 	 * @since 1.0.0
 	 * @access public
 	 *
-	 * @param array $args Optional. Nou used in remote source.
+	 * @param array $args Optional. Not used in remote source.
 	 *
 	 * @return array Remote templates.
 	 */
 	public function get_items( $args = [] ) {
-		$library_data = Api::get_library_data();
+		$force_update = $args['force_update'] ?? false;
```

Comment:
Maybe
`$force_update = is_bool( ( $args['force_update'] ?? false ) ) ?  $args['force_update'] : false;`
?

Reply:
Why?

Reply:
To make sure it's boolean and you do not pass something weird as the $force_update param.

Reply:
Done